src/core/client/stream/components/__snapshots__/Stream.spec.tsx.snap

```diff
@@ -1,7 +1,9 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`renders correctly 1`] = `
-<div>
+<div
+  className="Stream-root"
```

Comment:
Should classnames be uppercased? Is this something we can address via the css transformer?

Reply:
It’s fine to have camelcased classnames as long as it’s consistent. That aside it doesn’t really play much of a role in these snapshots as they are just for testing. It just helps you see where the css class names came from ${ComponentName}-${className} and is different from how it ends up in the browser which currently is just a hash